x-pack/test/functional/apps/endpoint/policy_list.ts

```diff
@@ -25,14 +24,16 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
       const policyTitle = await testSubjects.getVisibleText('policyViewTitle');
       expect(policyTitle).to.equal('Policies');
     });
-    it('shows policy count total', async () => {
+    // FIXME: Skipped until we can figure out how to load data for Ingest
+    it.skip('shows policy count total', async () => {
```

Comment:
perhaps just remove these tests? they'll be available in git history

Reply:
I'm thinking these will come right back in the not too far future.